### PR TITLE
test: bump 8.3 repo

### PR DESCRIPTION
**Pull request by ondrejbudai**
The previously used snapshot was deleted. This commit changes the override to
use Internal Snapshot 2.0, which should not be deleted in the near future.

### CODE REVIEW

File: test/internal-repos/rhel-8.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
     "x86_64": [
         {
-            "name": "RHEL 8.3 Nightly BaseOS",
-            "baseurl": "http://download.eng.rdu.redhat.com/rhel-8/nightly/RHEL-8/RHEL-8.3.0-20200611.n.0/compose/BaseOS/x86_64/os/",
+            "name": "RHEL 8.3 Internal Snapshot 2.0 BaseOS",
+            "baseurl": "http://download.eng.rdu.redhat.com/rhel-8/rel-eng/RHEL-8/RHEL-8.3.0-InternalSnapshot-2.0/compose/BaseOS/x86_64/os/",
```
Comment (atodorov) [inline test/internal-repos/rhel-8.json:5]: Why not use `latest-RHEL-8` symlink instead ?

Comment (ondrejbudai) [reply test/internal-repos/rhel-8.json:5]: I would prefer the environment to be as fixed as possible, hence the fixed snapshot.

**Comment by ondrejbudai** [comment]
According to Schutzbot, this fixes out 8.3 CI.

**Comment by major** [comment]
Thanks for cleaning this up, @ondrejbudai. It was on my to-do list before PTO. :(

**Pull request merged**